core/templates/pages/release-coordinator-page/navbar/release-coordinator-navbar.component.html

```diff
@@ -0,0 +1,174 @@
+<style>
+  .release-coordinator-navbar .nav > li > a:focus,
+  .release-coordinator-navbar .oppia-release-coordinator-page-dropdown-icon {
+    background-color: #00376d;
+    color: #fff;
+  }
+
+  .release-coordinator-navbar .oppia-navbar-profile-release-coordinator > li > a:hover,
+  .release-coordinator-navbar .oppia-navbar-profile-release-coordinator > li > a:focus,
+  .release-coordinator-navbar .oppia-navbar-profile-release-coordinator .open > a,
+  .release-coordinator-navbar .oppia-navbar-profile-release-coordinator .open > a:hover,
+  .release-coordinator-navbar .oppia-navbar-profile-release-coordinator .open > a:focus,
+  .release-coordinator-navbar .oppia-navbar-tabs-release-coordinator > li > a:hover,
+  .release-coordinator-navbar .oppia-navbar-tabs-release-coordinator > li > a.active,
+  .release-coordinator-navbar .oppia-release-coordinator-page-dropdown-icon {
+    background-color: #fff;
+    color: #00376d;
+  }
+  .oppia-navbar-tabs-release-coordinator .dropdown-toggle::after {
+    color: #fff;
+  }
+  .oppia-navbar-tabs-release-coordinator:hover .dropdown-toggle::after,
+  .oppia-navbar-tabs-release-coordinator:focus .dropdown-toggle::after {
+    color: #00376d;
+  }
+  .release-coordinator-navbar .oppia-navbar-release-coordinator {
+    top: 0;
+  }
+  .release-coordinator-navbar .navbar-container.oppia-navbar-container {
+    background-color: #00376d;
+  }
+  .release-coordinator-navbar .nav .dropdown:hover > .dropdown-menu > li > a.oppia-dropdown-item-text {
+    color: #00376d;
+  }
+  .release-coordinator-navbar .oppia-dropdown-toggle-icon {
+    margin-top: -1px;
+  }
+  @media screen and (max-width: 710px) {
+    .release-coordinator-navbar .oppia-release-coordinator-page-dropdown {
+      display: block;
+    }
+    .release-coordinator-navbar .oppia-hide-in-small-width-screens {
+      display: none;
+    }
+  }
+  @media screen and (min-width: 711px) {
+    .release-coordinator-navbar .oppia-release-coordinator-page-dropdown,
+    .release-coordinator-navbar .oppia-release-coordinator-page-dropdown-icon,
+    .release-coordinator-navbar .oppia-release-coordinator-page-dropdown-menu {
+      display: none;
+    }
+  }
+</style>
+<div class="release-coordinator-navbar">
+  <nav class="navbar navbar-default navbar-light oppia-navbar oppia-prevent-selection oppia-navbar-release-coordinator" role="navigation">
+    <div class="navbar-container oppia-navbar-container">
+      <div class="navbar-header protractor-test-navbar-header float-left">
+        <a class="oppia-navbar-brand-name oppia-transition-200 float-left" href="/community-library">
+          <picture>
+            <source type="image/webp" [srcset]="getStaticImageUrl('/logo/288x128_logo_white.webp')">
```

Comment:
Precompute this value and store it in class variable. Use that variable here. We should try to reduce having function calls in templates as it make the change detection cycles inefficient.

Reply:
I agree, done!